online/src/test/scala/ai/chronon/online/FetcherBaseTest.scala

```diff
@@ -141,4 +148,76 @@ class FetcherBaseTest extends MockitoSugar with Matchers {
     actualRequest.get.name shouldBe query.groupByName + "." + query.columnName
     actualRequest.get.keys shouldBe query.keyMapping.get
   }
+
+  // updateServingInfo() is called when the batch response is from the KV store.
+  @Test
+  def test_getServingInfo_ShouldCallUpdateServingInfoIfBatchResponseIsFromKvStore(): Unit = {
```

Comment:
Same as the other one, could you make it camel case only to be consistent with the naming convention? thank you!